edx_django_utils/plugins/docs/how_tos/how_to_enable_plugins_for_an_ida.rst

```diff
@@ -25,6 +25,8 @@ update:
 4. its setup to register PluginsConfig (for connecting Plugin Signals)
 ::
 
+.. note:: For a plugin that will appear in a single IDA, you could create constants like `ProjectType found in edx-platform`_. If the plugin is for many IDAs, we need to add a capability to this library with a global constant.
```

Comment:
This line results in weird formatting cause its in a list and after :: to signify code block

Reply:
Oops.  Thanks.  Fixed.